bot/seasons/evergreen/space.py

```diff
@@ -0,0 +1,154 @@
+import logging
+from datetime import datetime
+from random import randint
+from typing import Any, Dict, Optional
+from urllib.parse import urlencode
+
+from discord import Embed
+from discord.ext.commands import Cog, Context, command
+
+from bot.bot import SeasonalBot
+from bot.constants import Tokens
+
+logger = logging.getLogger(__name__)
+
+# NASA API base URL
+BASE_URL = "https://api.nasa.gov"
+NASA_IMAGES_BASE = "https://images-api.nasa.gov"
+EPIC_BASE_URL = "https://epic.gsfc.nasa.gov"
```

Comment:
I don't think the comment adds much to the already clear names here, so I'd remove it.

```suggestion
BASE_URL = "https://api.nasa.gov"
NASA_IMAGES_BASE = "https://images-api.nasa.gov"
EPIC_BASE_URL = "https://epic.gsfc.nasa.gov"
```

In addition, your constants do not really have a consistent naming scheme. The first one is simple named `BASE_URL` and does not name `NASA` while the second one is also a NASA url but suddenly has NASA in the name but not the `URL` suffix. `BASE_URL` also sounds like it's going to be used as the base for all URLs but all three are used as the base for different endpoints. 

I think something like `NASA_API_URL, NASA_IMAGES_URL, NASA_EPIC_URL` would be more consistent while still keeping their self-documenting nature even if they're some day imported by another module.